src/dynamodb.ts

```diff
@@ -20,16 +20,21 @@ export class Dynamo {
 
   /**
    * Returns the date formatted to CVS project standard.
+   *
    * @param {DateTime} date A Luxon Datetime.
-   * @returns {string}
+   * @returns {string} The formatted date
    */
-  public static toCVSDate(date: DateTime): string {
+  public static async toCVSDate(date: DateTime): Promise<string> {
```

Comment:
Does it need to be async?

Reply:
No

Reply:
can we make it sync if we don't need async operations for it please?